docs/tutorial/devtools-extension.md

```diff
@@ -45,6 +45,9 @@ The extension will be remembered so you only need to call this API once per
 extension. If you try to add an extension that has already been loaded, this method
 will not return and instead log a warning to the console.
 
+**Note:** The `BrowserWindow.addDevToolsExtension` API is deprecated. Here is an alternative.
```

Comment:
Instead of adding a note, it's probably more effective to just update the tutorial to use the correct new methods so let's do that instead.